chart/templates/controller/deployment.yaml

```diff
@@ -73,7 +73,7 @@ spec:
         {{- end }}
       containers:
         - name: csi-attacher
-          image: {{ tpl .Values.controller.image.csiAttacher.name . }}
+          image: {{ tpl .Values.controller.image.csiAttacher.name . | default (printf "%s:%s" .Values.controller.image.csiAttacher.repository .Values.controller.image.csiAttacher.tag) }}
```

Comment:
Could you make this a function in `chart/_common_images.tpl` so it can be reused across all images?

Reply:
I think this is no longer relevant after applying @jooola's comment.

If you think anything could be moved to a template function, please unresolve this conversation and let me know.